### PR TITLE
[Utility]Neutral Turret (want to be merged)

### PR DESCRIPTION
**Pull request by CoffeePot1206**
finish #156

### CODE REVIEW

**Comment by CoffeePot1206** [comment]
[README.txt](https://github.com/Yao-class-cpp-studio/battle_game/files/10398269/README.txt)
Features, bugs are included in the README. (Why cannot I attach .md file...)

File: src/battle_game/core/units/neutral_turret.h [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  void Fire();
+  [[nodiscard]] const char *UnitName() const override;
+  [[nodiscard]] const char *Author() const override;
+
+ private:
+  float turret_rotation_{0.0f};
+  uint32_t fire_count_down_{0};
+  float range_{5.0f};
+  glm::vec2 scale_{1.0f};
+  glm::vec2 findEnermy();
```
Comment (LazyJazz) [inline src/battle_game/core/units/neutral_turret.h:24]: BigCamelCase

**Review by LazyJazz — CHANGES_REQUESTED** [review]

**Comment by CoffeePot1206** [comment]
[README.txt](https://github.com/Yao-class-cpp-studio/battle_game/files/10407748/README.txt)

出生点的bug已经修复。

**Review by LazyJazz — APPROVED** [review]
LGTM!

**Pull request closed without merging**